### DISCUSSION TITLE
NoM - Phase 1 Roadmap tweet

### DISCUSSION THREAD

**Discussion by sugoibtc** [coinselor/twitter#17] [Tweet Ideas]
🚀 Embarking on NoM Phase 1: A leap towards a global tech revolution! 

Roadmap highlights:

- Merge-mining with #BTC
- Narwhal & Tusk for 100k+ TPS
- ZVM Extension chain: WASM/EVM ready
- s y r i u s mobile wallet
- Ledger support

Decrypting the future, one innovation at a time.

**Comment by sugoibtc** [comment]
Image still needs to be added in the final draft.

**Comment by coinselor** [comment]
NoM Phase 1: A leap towards a global tech revolution!

Roadmap highlights:

- Peer to peer merged mining with #BTC and #XMR
- Narwhal & Tusk for 100k+ TPS
- Zenon Virtual Machine: WASM and EVM ready
- s y r i u s mobile wallet
- Ledger support

Follow for detailed breakdowns coming soon!


Here's my suggestion: just added peer to peer and monero, removed extension chain wording and made it fit within the X character limit.

Changed CTA for a follow + hint at more.

**Comment by coinselor** [comment]
```
---
media:
  - file: roadmap.png
    alt: An infographic showing a table of the roadmap items for NoM Phase 1.
---
NoM Phase 1: A leap towards a global tech revolution!

Roadmap highlights:

- Peer to peer merged mining with #BTC and #XMR
- Narwhal & Tusk for 100k+ TPS
- Zenon Virtual Machine: WASM and EVM ready
- s y r i u s mobile wallet
- Ledger support

Follow for detailed breakdowns coming soon!
```
